src/index.ts

```diff
@@ -59,6 +59,28 @@ enum CloudWatchLogTypes {
 }
 
 const SQUID_ACCESS_LOG_FORMAT_REGEX = /^\d+\.\d{3} /
+const SERVICES = [
+  'adminusers',
+  'cardid',
+  'concourse-monitoring',
+  'concourse-web',
+  'connector',
+  'egress',
+  'frontend',
+  'ledger',
+  'notifications',
+  'products',
+  'publicapi',
+  'publicauth',
+  'pushgateway',
+  'selfservice',
+  'toolbox',
+  'tooling-pactbroker',
+  'tooling-stubs',
+  'webhooks',
+  'webhooks-egress'
+]
```

Comment:
If we add a new service, it may not be added here, and the logs will likely go unnoticed for a while.

Can we replace `environment name`, `-alb`, `pay-cd-`, `deploy-tooling` and use the remaining for the service name?

Reply:
yeah I put this back in draft because I had the same thought. I agree its better to strip out expected prefix and suffix and then send the rest 👍 I'll make the change.